tools/release/patch/v0.14.1/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM amazon/aws-otel-collector:v0.14.0
+
+COPY config/ecs/ecs-*.yaml /etc/ecs/
```

Comment:
How to we plan on handling patches that are more than just a config change?

Reply:
We would likely need to fully rebuild the image and follow our normal release steps. This is a bit of a special case.

Reply:
This is a special case. But we should have a patch building process which should include its own workflow.